src/components/forms/promocode-form.js

```diff
@@ -562,6 +562,12 @@ class PromocodeForm extends React.Component {
         entity.type = null;
         entity.allowed_ticket_types = [];
         entity.ticket_types_rules = [];
+        entity.quantity_available = '';
```

Comment:
i do think that we need to review 
https://github.com/fntechgit/summit-admin/blob/d40cf28f3b37536c6ae0bca6d9ae84fa07c74b2b/src/reducers/promocodes/promocode-reducer.js#L49